Sources/SwiftSyntax/Convenience.swift

```diff
@@ -45,3 +45,56 @@ extension MemberAccessExprSyntax {
     )
   }
 }
+
+extension EnumCaseParameterSyntax {
+
+  /// Creates an `EnumCaseParameterSyntax` with a `firstName`, and automatically adds a `colon` to it.
+  ///
+  /// - Parameters:
+  ///   - leadingTrivia: Trivia to be prepended to the leading trivia of the node’s first token. If the node is empty, there is no token to attach the trivia to and the parameter is ignored.
+  ///   - colon: If the parameter has a label, the colon separating the label from the type. SInce `firstName` is not nil, the default `colon` will be set to `TokenSyntax.colonToken()`.
+  ///   - type: The parameter's type.
+  ///   - defaultValue: If the parameter has a default value, the initializer clause describing the default value.
+  ///   - trailingComma: If the parameter is followed by another parameter, the comma separating them.
+  ///   - trailingTrivia: Trivia to be appended to the trailing trivia of the node’s last token. If the node is empty, there is no token to attach the trivia to and the parameter is ignored.
```

Comment:
Awesome that you've added a doc comment to this `init`! I noticed it's the same as the one in the original `init`. Given that the original comment is auto-generated by the [`CodeGeneration` package](https://github.com/apple/swift-syntax/tree/main/CodeGeneration), it could be updated fairly easily, making this one obsolete pretty quickly. I'm wondering if there's a way to address it. Though, to be honest, coming up with an advanced solution might just be overengineering the whole thing. What do you think?

Reply:
You're absolutely right re: getting out of sync. 

> Though, to be honest, coming up with an advanced solution might just be overengineering the whole thing.

When I looked at that whole issue, I thought that if we really wanted to, there must be a way to tweak code generation to generate multiple convenience initializers as well. That would probably take some time, but could be quite elegant.

The most simple way out of this is to make a shorter comment and link to the generated method docs for details on other parameters.

Reply:
For now, my suggestion would be to keep the part that you wrote by hand and add a `- SeeAlso:` section that links to the underlying initializer. 

As we write more of these initializers, maybe a pattern will emerge and we decide to generate the initializers, in which case we can also generate the section. But for now, that would just be over-engineering IMO.

Reply:
Done!